src/pages/_app.tsx

```diff
@@ -18,14 +16,14 @@ export const App = ({ Component, pageProps }) => {
           --inter-font: ${inter.style.fontFamily};
         }
       `}</style>
-      <AuthWalletContextProvider>
-        <ChakraProvider theme={theme}>
-          <PageHead />
-          <Layout>
-            <Component {...pageProps} />
-          </Layout>
-        </ChakraProvider>
-      </AuthWalletContextProvider>
+      {/* <AuthWalletContextProvider> */}
```

Comment:
By commenting `AuthWalletContextProvider` and `Layout` out, the app is able to render the home page. I would suggest to read Next.js 13 on layouts and context handling..  https://beta.nextjs.org/docs/upgrade-guide